pkg/tiles/tile.go

```diff
@@ -0,0 +1,85 @@
+package tiles
+
+import (
+	buildings "github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/buildings"
+	connectionMod "github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/connection"
+	featureMod "github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/feature"
+)
+
+/*
+Immutable object
+*/
+type Tile struct {
+	Features  []featureMod.Feature
+	HasShield bool
+	Building  buildings.Bulding
+}
+
+func (tile *Tile) Cities() [][]connectionMod.Side {
```

Comment:
Hmm, wouldn't it be simpler to just return `[]Feature`? Same question for `Roads()` and `Fields()` as well.

Reply:
changed